csrankings-d.csv

```diff
@@ -1088,6 +1088,7 @@ Dimitrios Pezaros,University of Glasgow,http://www.dcs.gla.ac.uk/~dp,7_7cAlgAAAA
 Dimitrios Rafailidis,Maastricht University,http://www.drafail.eu,0aMmyHgAAAAJ
 Dimitrios S. Kolovos,University of York,https://www-users.cs.york.ac.uk/~dkolovos,aRd1i6UAAAAJ
 Dimitrios S. Nikolopoulos,Virginia Tech,https://www.dsniko.net,-nzj5UYAAAAJ
+Dimitrios Skarlatos 0002,CMU,https://www.cs.cmu.edu/~dskarlat,T_ZmDEAAAAAJ^M
```

Comment:
CMU is called "Carnegie Mellon University" in the database. Please fix all entries and remove spurious carriage return (^M) in this line.

Reply:
Fixed. Thanks!